WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostDetailFragment.kt

```diff
@@ -663,6 +681,20 @@ class ReaderPostDetailFragment : ViewPagerFragment(),
                 .newInstance()
                 .show(childFragmentManager, JetpackPoweredBottomSheetFragment.TAG)
         }
+
+        viewModel.reloadFragment.observeEvent(viewLifecycleOwner) {
+            if (isAdded) {
+                // TODO thomashortadev this works but looks a bit bad since the post and images are reloaded
```

Comment:
I couldn't find a way around it so far, so I left that TODO to figure out in the future, though I can't think of a better solution, so it might be worth just removing the TODO for now.